$MirOS: ports/www/lynx/patches/patch-LYMessages_en_h,v 1.1 2010/12/23 20:25:30 tg Exp $

	Fix spelling; suggested for upstream inclusion.

--- LYMessages_en.h.orig	Mon Feb 13 02:45:53 2012
+++ LYMessages_en.h	Sat Mar  3 18:20:20 2012
@@ -606,7 +606,7 @@
 #define ERROR_UNCOMPRESSING_TEMP gettext("Error uncompressing temporary file!")
 #define UNSUPPORTED_URL_SCHEME gettext("Unsupported URL scheme!")
 #define UNSUPPORTED_DATA_URL gettext("Unsupported data: URL!  Use SHOWINFO, for now.")
-#define TOO_MANY_REDIRECTIONS gettext("Redirection limit of 10 URL's reached.")
+#define TOO_MANY_REDIRECTIONS gettext("Redirection limit of 10 URLs reached.")
 #define ILLEGAL_REDIRECTION_URL gettext("Illegal redirection URL received from server!")
 #define	SERVER_ASKED_FOR_REDIRECTION \
  gettext("Server asked for %d redirection of POST content to")
